.github/workflows/pull-request-checks.yaml

```diff
@@ -40,6 +40,42 @@ jobs:
           env PATH=$PATH:`pwd`/src/solvers make -C regression/cbmc test-cprover-smt2
           make -C jbmc/regression test
 
+  check-ubuntu-20_04-make-clang:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Fetch dependencies
+        env:
+          # This is needed in addition to -yq to prevent apt-get from asking for
+          # user input
+          DEBIAN_FRONTEND: noninteractive
+          TESTPL_JOBS: 4
+        run: |
+          sudo apt-get install -yq gcc gdb g++ maven jq flex bison libxml2-utils cpanminus
```

Comment:
:grey_question: Do we need `gcc gdb g++` for a `clang` build?

Reply:
I thought about it - I opted for keeping the job's environment in line with the `linux+make+gcc` one.

Though these are arguably not very useful given that they are preinstalled in Github runners. But I suggest this stays for now and we have a separate discussion on whether we should remove them from the other build as well.